### ISSUE TITLE
Uncaught (in promise) TypeError: tab.sendMessage is not a function

### ISSUE THREAD

**Issue by SkyLeach** [glacambre/firenvim#1045]
- OS Version: win x86-64
- Browser Version:  Firefox 88.0
- Browser Addon Version: 0.2.6
- Neovim Plugin Version: 0.2.6
- What I tried to do: Get it (ff/chrome/neovim plugin) working
- What happened: Black unresponsive rectangle.

I managed to get as far as the error message listed in this issue's subject line before I ran out of time to spend on this (for now at least).

Maybe later I will get time to clone the repo and change bg.js line 134 where I'm guessing it should read tabs.sendMessage (but I'm not really sure).

![image](https://user-images.githubusercontent.com/940228/118423126-50158480-b692-11eb-8349-e6614d77aac8.png)

**Comment by glacambre** [comment]
Hi, thanks for the report and for going through the troubleshooting guide! Firenvim doesn't have a "bg.js" file, so I don't think the "tab.sendMessage is not a function" error is comming from Firenvim. The source of that error is probably another add-on.

However, thanks to the screenshot of the background console, I can see that your neovim instance is printing error messages on startup. Firenvim can't work if neovim emits any messages - the reason for that is that communication between neovim and firefox must happen with a specific format.

So what I would recommend is trying to [run the firenvim script](https://github.com/glacambre/firenvim/blob/master/TROUBLESHOOTING.md#make-sure-the-firenvim-script-has-been-created) from the command line to see what the error messages are, fix them, and then try to start Firenvim again :).

**Comment by SkyLeach** [comment]
(time issue, will return to this this week)

**Comment by glacambre** [comment]
Closing this issue as it's been awaiting feedback from its author for more than a month. If you're still experiencing this issue, please go through the instructions I gave in my previous message, tell me what the result is and I'll re-open :)

**Comment by SkyLeach** [comment]
Sorry, I've been extraordinarily busy.  The problem I was experiencing was so convoluted I gave up for a while.  After retooling almost my entire system, I tried firenvim again and it worked.  I don't know what the root cause was, but I'm using it right now.  TY, and sorry for getting lost in the weeds and not coming back for so long.

![image](https://user-images.githubusercontent.com/940228/127887714-b903a3f7-c683-47fb-a22e-eae470e45eda.png)
